mount/mount_windows.go

```diff
@@ -55,6 +55,46 @@ func (mounter *Mounter) Mount(source string, target string, fstype string, optio
 	return mounter.MountSensitive(source, target, fstype, options, nil /* sensitiveOptions */)
 }
 
+//mountNfs uses powershell command "net use" to connect NFS share
+func mountNfs(source, target string, options []string) error {
+	mountCmd := "powershell /c net use " + source
+	output, err := exec.Command(mountCmd, options...).CombinedOutput()
+	if err != nil {
+		klog.Errorf("NFS mount failed: %v, source(%q) output: %q", err, source, target, string(output))
```

Comment:
Could there be any sensitive output like credentials in the error message?

Reply:
I tried out this command, and password information is not in error message.

Reply:
Can we return this as the error message and leave logging to the caller?

Reply:
removed